### DISCUSSION TITLE
Why can't I use two class selectors?

### DISCUSSION THREAD

**Discussion by Peter-Dobbins** [community/community#27312] [Other Feature Feedback, Questions, & Ideas] [answered]
Can anybody tell me why the text box is not pink?

<img width="1440" alt="Screenshot 2022-07-27 at 15 31 03" src="https://user-images.githubusercontent.com/109819737/181275087-5d481d6d-6361-48bf-9e9f-5cebd9ad4858.png">

**Comment by airtower-luna** [comment, marked as answer]
You can't have two `class` attributes in one element, but you can list multiple classes in one attribute.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class:

> The class [global attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes) is a space-separated list of the case-sensitive classes of the element.

**Reply by Peter-Dobbins** [reply]
Thank you!
